src/components/event/EventActiveStatus/RenderEventButton.js

```diff
@@ -3,10 +3,11 @@ import IconButton from "@material-ui/core/IconButton";
 import Tooltip from "@material-ui/core/Tooltip";
 import propTypes from "prop-types";
 
+const MIDDLE = "middle";
```

Comment:
Extract this constant variable into separate file with other constants

Reply:
done